server/util.go

```diff
@@ -54,6 +55,7 @@ func AddCommands(
 		sdkserver.VersionCmd(),
 		tmcmd.ResetAllCmd,
 		tmcmd.ResetStateCmd,
+		sdkserver.BootstrapStateCmd(opts.AppCreator),
```

Comment:
you mean this isn't included on the hub already? @VictorTrustyDev

Reply:
It wasn't there at time of wring this, Aug last year.
Idk now.